rmgpy/molecule/molecule.py

```diff
@@ -669,6 +675,9 @@ class Molecule(Graph):
     `symmetryNumber`        ``int``     The (estimated) external + internal symmetry number of the molecule
     `multiplicity`          ``int``     The multiplicity of this species, multiplicity = 2*total_spin+1
     `props`                 ``dict``    A list of properties describing the state of the molecule.
+    `InChI`                 ``str``     A string representation of the molecule in InChI
+    `atoms`                 ``list``    A list of Atom objects in the molecule
```

Comment:
In the paragraph above this table, it says that the `atoms` and `bonds` attributes are aliases for the `vertices` and `edges attributes. If you're going to list `atoms` in the table, I think you can remove it from the text. Also, there is no `bonds` bonds attribute (or `edges` for that matter), so I think you should also correct that.